index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to Viable Groups</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="navbar-black">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+            <div class="menu">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                
+                <ul class="header-menu">
+                    <li class="header-menu-item"><a href="#work">English</a></li>
+                    <li class="header-menu-item"><a href="#about">My Page</a></li>
+                    <li class="header-menu-item"><a href="#contact">Login</a></li>
+                </ul>
+            </div> 
+        </nav> 
+        <nav class="navbar-white">
+            <div>
+                <img src="images/vg-logo.png" alt="VG Logo">
+            </div>
+            <ul class="header-white-menu">
+                <li class="header-white-menu-item"><a href="#about">About</a></li>
```

Comment:
- [ ] When I click on `About` link, a new page should open with a `footer` and its own `navbar`, but instead I am taken to the different section of the same page with `#about` id. I noticed that in project you have only 1 html index file. Please kindly add `about.html` file as well and make sure that the About link takes to `about.html` page. 
- [ ] While writing `about.html` please try to stick  to provided [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).